src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/TextViewWindow_OutOfProc.Verifier.cs

```diff
@@ -30,12 +31,21 @@ public Verifier(TTextViewWindow textViewWindow, VisualStudioInstance instance)
                 FixAllScope? fixAllScope = null,
                 bool blockUntilComplete = true)
             {
+                using var cancellationTokenSource = new CancellationTokenSource(Helper.HangMitigatingTimeout);
+
                 var expectedItems = new[] { expectedItem };
-                CodeActions(expectedItems, applyFix ? expectedItem : null, verifyNotShowing,
-                    ensureExpectedItemsAreOrdered, fixAllScope, blockUntilComplete);
+
+                bool applied;
+                do
+                {
+                    cancellationTokenSource.Token.ThrowIfCancellationRequested();
+
+                    applied = CodeActions(expectedItems, applyFix ? expectedItem : null, verifyNotShowing,
+                        ensureExpectedItemsAreOrdered, fixAllScope, blockUntilComplete) ?? true;
```

Comment:
can you explain the reason you want the `??` true here?

Reply:
No longer using `??`